wcfsetup/setup/db/install.sql

```diff
@@ -1839,6 +1819,18 @@ CREATE TABLE wcf1_user_rank (
 	hideTitle TINYINT(1) NOT NULL DEFAULT 0
 );
 
+DROP TABLE IF EXISTS wcf1_user_session;
+CREATE TABLE wcf1_user_session (
+	sessionID CHAR(40) NOT NULL PRIMARY KEY,
```

Comment:
Test whether a `BIGINT` `auto_increment` column for the `PRIMARY KEY` with `sessionID` being a `UNIQUE KEY` improves `INSERT` and `UPDATE` performance.

Reply:
It most likely would improve `INSERT` performance, but tank `SELECT` (and probably also `UPDATE`) performance, because of the additional indirection. It's not entirely clear which type of query dominates here, because that depends on the cookies the user-agent sends. The safest option should be using what worked in the past: `sessionID` being the `PRIMARY KEY`.